src/overlays/actors/ovl_En_Zl2/z_en_zl2.c

```diff
@@ -5,6 +5,9 @@
  */
 
 #include "z_en_zl2.h"
+#include "vt.h"
+
```

Comment:
in the rest of the codebase we do <vt.h> as opposed to quotes. not sure if one is more correct over the other

Reply:
oh huh, I'll change it then. must have just forgotten